services/actions-handler/go.mod

```diff
@@ -4,7 +4,7 @@ go 1.18
 
 require (
 	github.com/cheshir/go-mq v1.2.0
-	github.com/uselagoon/machinery v0.0.7
+	github.com/uselagoon/machinery v0.0.8-0.20221129235134-66a9c5a0ca7b
```

Comment:
This is a reference to a specific commit? Is that a datestamp 2022-11-29? Is there a stable release we could reference instead?

This is outside my knowledge area, I'll defer entirely to your judgement.

Reply:
Yes, this is a reference to a specific commit, and that is the timestamp and commit sha. I can bump this to a stable release but the feature that it implements isn't in the Lagoon API until this PR is merged.

Reply:
ah, right. `machinery` is the api client, and if you try to query a non-existent field graphql will error. I'd be interested in figuring out a better way to handle this chicken/egg problem, but not right now